tests/system/Commands/Utilities/NamespacesTest.php

```diff
@@ -54,6 +54,9 @@ public function testNamespacesCommandCodeIgniterOnly()
             | Tests\Support | ROOTPATH/tests/_support | Yes    |
             +---------------+-------------------------+--------+
             EOL;
+        $expected = str_replace('ROOTPATH/', 'ROOTPATH' . DIRECTORY_SEPARATOR, $expected);
+        $expected = str_replace('APPPATH/', 'APPPATH' . DIRECTORY_SEPARATOR, $expected);
+        $expected = str_replace('tests/_support', 'tests' . DIRECTORY_SEPARATOR . '_support', $expected);
```

Comment:
Can't you remove `DIRECTORY_SEPARATOR`?

Reply:
What would the alternative be?

Reply:
For example, normalize the command output. I mean always outputs paths with `/`.

Reply:
Got it! Let's see what Paul comes back with.

Reply:
Used regex to extract the middle text. Thanks, chatGPT 😂

Reply:
I don't think using complicated regex is the best, but it is better than the first code.

I am also not sure that normalizing the separator in output to `/` in Windows is the best thing to do.

Anyway, thanks Paul.